### PR TITLE
Add uboot CVEs

### PR DESCRIPTION
**Pull request by sciencemanx**

### CODE REVIEW

File: uboot-cve-2019-13103-13106/README.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+Once inside the docker run:
+
+```bash
+cd /workdir
+/mayhem/u-boot-2019.07-rc4/u-boot -c "host bind 0 /PATH/TO/CRASH ; ls host 0"
+And replace the path to crash with any crash in the crash file.
+```
+
+## POC
```
Comment (dbrumley) [inline uboot-cve-2019-13103-13106/README.md:54]: Can you add a new section that mentions we had to change AFL for it to repro? That caught me by surprise looking through the diff, and I could imagine someone missing that.

File: uboot-cve-2019-13103-13106/README.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
+# uboot CVE 2019-13103 to 2019-13106
+
+This repo replicates finding [CVE-2019-13103](https://nvd.nist.gov/vuln/detail/CVE-2019-13103) and following CVE's. Many crashes are found including floating point errors and stack corruption bugs (Max [CVSS Score](https://nvd.nist.gov/vuln-metrics/cvss): 8.3).
+
+Das U-Boot is an open-source bootloader for embedded devices. Usually, it is
+the first thing the CPU runs, and it will boot into some form of linux from
+storage media. To fuzz it, we use the sandbox feature of u-boot, which
+compiles it as a linux userspace program so that features can be tested off
+of the embedded device. We stress the ext4 code by listing files.
+
```
Comment (dbrumley) [inline uboot-cve-2019-13103-13106/README.md:12]: nice job on this section!

File: uboot-cve-2019-13103-13106/README.md (hunk 1 above)
Comment (sciencemanx) [reply uboot-cve-2019-13103-13106/README.md:54]: The change is actually to the uboot code base for deferred fork server with afl fast clang

Comment (sciencemanx) [reply uboot-cve-2019-13103-13106/README.md:54]: (will still add note in readme though)

**Pull request merged**